docs/source/installation.rst

```diff
@@ -350,20 +350,36 @@ The building command depends on the architecture of the Python interpreter,
 The above commands assume that you have the Python installation folder in your
 PATH environment variable.
 
-You will need `Build Tools for Visual Studio 2017
-<https://visualstudio.microsoft.com/downloads/>`_.
+Installing Build Tools for Visual Studio 2022
+"""""""""""""""""""""""""""""""""""""""""""""
+You will need Build Tools for Visual Studio 2022
 
 .. warning::
-	You DO NOT need to install Visual Studio 2019.
-	You only need the "Build Tools for Visual Studio 2019",
-	under "All downloads" -> "Tools for Visual Studio 2019".
+	You DO NOT need to install Visual Studio 2022.
+	You only need the "Build Tools for Visual Studio 2022"
+
+1. Follow the `link <https://visualstudio.microsoft.com/downloads/>`_.
+2. Scrolls down to "Tools for Visual Studio 2022" and click the dropdown,
+3. Download and install "Build Tools for Visual Studio 2022"
+4. Open the Visual Studio installer
+5. Tick "Desktop Development with C++"
+6. In the panel to the write make sure "MSVC v143 VS 2022 C++ x64/x86 build tools (Latest)" is selected
+7. Click install
+
+.. images:: images/visual_installer_selection.png
 
 For 64-bit Python, configure the build environment with:
```

Comment:
may need some more instructions that this should be typed in command line or power shell. The current formulation "configure the build environment" is vague on what exactly to do. It sounds like an instruction, but in fact it's a summary, that ambiguity should be resolved.